app/javascript/leaflet/viewers/index_map.js

```diff
@@ -0,0 +1,176 @@
+import { geoJson, circleMarker } from "leaflet";
+import LeafletViewerMap from "./map.js";
+
+export default class LeafletViewerIndexMap extends LeafletViewerMap {
+  onLoad() {
+    super.onLoad();
+    this.map.addLayer(this.selectBasemap());
+
+    if (this.available) {
+      this.addPreviewLayer();
+      this.addControls();
+    } else {
+      this.addBoundsOverlay(this.options.bbox);
+    }
+  }
+
+  indexMapInfoTemplate = (data) => `
```

Comment:
I think this can be split into a function. I think you can create a function call docMetadata that takes a field

def docMetadata(field, fieldlabel, link=false) {
if (!field) { return }
var html = `<dt class="col-sm-3">${fieldlabel}:</dt>`
if (link){
html += `<dt class="col-sm-3"><a href="{{field}}">{{field}}</a></dt>`
} else {
html += `<dd class="col-sm-9">{{field}}</dd>`
}